refinery/ui/source/js/file-browser/services/factory.js

```diff
@@ -293,7 +307,9 @@
         pinnedLeft: true,
         cellTemplate: cellTemplate,
         visible: isToolSelected,
-        enableCellEdit: false
+        cellEditableCondition: false,
+        cellClass: function (grid) { return addNonEditCellClass('Internal', grid);},
+        headerCellClass: function (grid) {return addNonEditCellClass('Internal', grid);},
```

Comment:
Maybe define `function (grid) {return addNonEditCellClass('Internal', grid);}` once and then reuse it?